redaxo/src/addons/install/lang/en_gb.lang

```diff
@@ -70,5 +70,5 @@ install_settings_error = Settings not saved, check file permissions for: {0}
 
 install_header_function = Function
 
-install_clear = Clear
+install_clear = Reset input
```

Comment:
Wobei ich mir nicht sicher bin, ob ich diese Bezeichnungen überhaupt besser finde.
Gerade hier im Englischen finde ich ein einfaches "Clear" eigentlich tendenziell besser.

Reply:
Ich habe es hier übernommen: 
https://github.com/redaxo/redaxo/blob/375994076212d7289eece4292d7f21eacdb2d4ed/redaxo/src/core/lang/en_gb.lang#L334

Würde jetzt mal so belassen und wie @schür sagt, das Ganze in einem späteren PR noch mal angehen.